recipes/nudup/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: nudup
+  version: "2.2_post2016104"
+
+source:
+  git_rev: 740d9fe439dd8917605a56483a8796b377eb24c6
+  git_url: https://github.com/nugentechnologies/nudup.git
+
+build:
+  number: 0
+  skip: True # [win]
+  skip: True # [py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - samtools >=0.1.18
```

Comment:
I looked for that in the docs, but couldn't find it. Thanks for pointing it out.